libs/tokens/src/migrations/tokensLegacyStateMigration.ts

```diff
@@ -67,12 +67,14 @@ function migrateLegacyTokenLists() {
     [SupportedChainId.MAINNET]: [],
     [SupportedChainId.GNOSIS_CHAIN]: [],
     [SupportedChainId.GOERLI]: [],
+    [SupportedChainId.SEPOLIA]: [],
   }
 
   const tokenListsState: TokenListsState = {
     [SupportedChainId.MAINNET]: {},
     [SupportedChainId.GNOSIS_CHAIN]: {},
     [SupportedChainId.GOERLI]: {},
+    [SupportedChainId.SEPOLIA]: {},
```

Comment:
More candidate places to become dynamic.

Reply:
But also, this should probably be removed by now, right?
Of course, not part of this PR.

Reply:
I would wait a couple months more before deleting the migration